src/components/BreadCrumb/BreadCrumb.component.jsx

```diff
@@ -7,7 +7,10 @@ function BreadCrumb(props) {
   const { authenticated } = useAuth();
 
   return (
-    <ul className={'breadcrumb ' + props.className}>
+    <ul
+      data-testid={props['data-testid']}
```

Comment:
Great! Using `data-testid`  is better than using `id` or `classname` as selectors, also, there is a way to get rid of these attributes if you don't want to have them in production https://dpericich.medium.com/removing-data-test-attributes-from-react-production-dom-5ea4ea018acc